app/operations/placements/change_status.rb

```diff
@@ -12,7 +13,11 @@ def call
 
     return Success(placement) if old_status == new_status
 
-    placement.status = new_status
+    disqualify_reason = DisqualifyReason.find_by(id: disqualify_reason_id)
```

Comment:
Nitpick: do we need to make request to database if new status is not disqualified?